STP/Tappable.swift

```diff
@@ -24,16 +24,36 @@ public extension Tappable where Self:UIView {
         let gestureRecognizer = UILongPressGestureRecognizer { [unowned self] (recognizer) -> Void in
             let tap = recognizer as! UILongPressGestureRecognizer
 
+            func isInFrame() -> Bool {
+                let location = recognizer.locationInView(self)
+                switch location {
+                case _ where location.x < 0 - self.frame.size.width / 2,
```

Comment:
Is there any reason that you didn't use `CGRectContainsPoint(frame, location)` here?